File: condicionales/02.cpp
// Dado un número natural, determinar si es cero, par o impar

#include "iostream"
using namespace std;

int main()
{
    int num;
    cout << "Inserte un numero: ";
    cin >> num;
    if (num == 0)
    {
        cout << "El numero " << num << " es CERO";
    }
    else if (num % 2 == 0)
    {
        cout << "El numero " << num << " es PAR";
    }
    else
    {
        cout << "El numero " << num << " es IMPAR";
    }
    return 0;
}

File: condicionales/02.py
num = int(input('Inserte un numero: '))
if num == 0:
    print(f'El numero {num} es CERO')
elif num % 2 == 0:
    print(f'El numero {num} es PAR')
else:
    print(f'El numero {num} es IMPAR')